eb_hooks.py

```diff
@@ -744,6 +762,7 @@ def inject_gpu_property(ec):
 PRE_CONFIGURE_HOOKS = {
     'at-spi2-core': pre_configure_hook_atspi2core_filter_ld_library_path,
```

Comment:
This looks good, should we remove the hooks that work around this then since they are no longer required (and rebuild the affected packages?)

Reply:
Yep, I'll remove those and try to do rebuilds for the affected packages, just wanted to make sure that GObject-Introspection itself builds without any issues.